client/src/components/LeafletOsmMap/index.js

```diff
@@ -3,8 +3,7 @@ import style from "./style";
 import MapPane from './MapPane';
 import Search from '../../components/Search';
 import SearchResults from '../../components/SearchResults';
-import { makeRequest } from '../../js/server-requests-utils';
```

Comment:
Can you add this back? This is needed internally for this module to push requested saved pins into the database.

Reply:
Sorry, not sure where I lost it...coding on two different computers and still getting accustomed to Github.  

Task Completed.